core/src/main/java/org/elasticsearch/common/geo/builders/ShapeBuilder.java

```diff
@@ -728,7 +729,17 @@ public static ShapeBuilder parse(XContentParser parser, GeoShapeFieldMapper shap
             Distance radius = null;
             CoordinateNode node = null;
             GeometryCollectionBuilder geometryCollections = null;
-            Orientation requestedOrientation = (shapeMapper == null) ? Orientation.RIGHT : shapeMapper.fieldType().orientation();
+
+            Map<String, Object> parseOptions = new HashMap<>();
```

Comment:
@colings86 absolutely. I debated this myself.  In the end I made the decision to put it in now for the flexibility of adding future options by just adding them to the hashmap. Since the overhead of the map is greater than the current parameters I can take it out and add it later when needed.